chapeau.py

```diff
@@ -122,6 +120,55 @@ def parse_function(data):
 
 routing_dictionary = {}
 
+def add_route(path, handler):
+	'''
+		params:
+			path:   <str>
+			handler: <function>
+
+		Adds the path as a key to the routing dictionary and maps
+		it to the function assigned to the path 
+	'''
+	if not isinstance(path, str):
+		raise TypeError
+	routing_dictionary[path] = handler
+
+def route(path):
+	'''
+		Decorator function
+		params:
+			path: <str>
+		Example:
+			@route('/user/leta')
+			def user_leta(request, client):
+				chapeau.render(client, '/path/to/file', *args, **kwargs)
+	'''
+	def route_decorator(func):
+		@wraps(func)
+		def wrapper(*args, **kwargs):
+			add_route(path, func)
+			return func(*args, **kwargs)
+		return wrapper
+	return route_decorator
+
+## Example usage of the route decorator
+##
+## @route('/hello')
```

Comment:
this line should be @chapeau.route('/hello'), however I'm not sure if this first example should be included in the docstring.  If I'm mistaken, please correct me, but I'm not sure if we can this example makes sense with Chapeau.  As in, will Chapeau actually work if we route to a function that just returns raw html?